adcm_aio_client/core/objects/cm.py

```diff
@@ -110,3 +116,56 @@ def get_own_path(self: Self) -> Endpoint:
 
 class ComponentsNode(NonPaginatedChildAccessor[Service, Component, None]):
     class_type = Component
+
+
+class ConfigsNode(PaginatedChildAccessor): ...
+
+
+class HostProvidersNode(PaginatedChildAccessor): ...
+
+
+class Host(InteractiveChildObject[Cluster | None]):
+    @property
+    def id(self: Self) -> int:
+        return int(self._data["id"])
+
+    @property
+    def name(self: Self) -> str:
+        return str(self._data["name"])
+
+    @property
+    def description(self: Self) -> str:
+        return str(self._data["description"])
+
+    @property
+    def status(self: Self) -> ADCMEntityStatus:
+        return ADCMEntityStatus(self._data["status"])
```

Comment:
Change pls this method on `get_status`. See example in `Cluster` object

Reply:
changed